vignettes/mcstate2.Rmd

```diff
@@ -0,0 +1,142 @@
+---
+title: "Introduction to mcstate2"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Introduction to mcstate2}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+This vignette is an introduction to some of the ideas in `mcstate2`.  As the package is not ready for users to use it, this is more a place for us to jot down ideas about what it will do.
+
+We're going to change the package name in future (dropping the `2` at least, but possibly also changing it entirely).  It's not yet clear to us if this will be a package that users will use or something that the rest of our stack will target.
```

Comment:
```suggestion
We're going to change the package name in the future (dropping the `2` at least, but possibly also changing it entirely).  It's not yet clear to us if this will be a package that users will use or something that the rest of our stack will target.
```